src/remote-control/process-update-delegate.js

```diff
@@ -194,23 +343,58 @@ export default class ProcessUpdateDelegate {
      * @returns {void}
      */
     _executeIfInMeeting(command, data) {
+        const meetingApi = this._getMeetingApiIfInMeeting();
+
+        if (meetingApi) {
+            meetingApi.executeCommand(command, data);
+
+            return;
+        }
+
+        logger.error('Failed to execute command.');
+    }
+
+    /**
+     * Returns the instance of {@code JitsiMeetExternalApi} for an active
+     * meeting.
+     *
+     * @private
+     * @returns {JitsiMeetExternalApi|null}
+     */
+    _getMeetingApiIfInMeeting() {
         const state = this._store.getState();
         const meetingApi = getMeetingApi(state);
 
         if (!meetingApi) {
-            logger.error('Tried to execute command without meeting api.');
-
-            return;
+            return null;
         }
 
         const { inMeeting } = getInMeetingStatus(state);
 
         if (!inMeeting) {
-            logger.error('Tried to execute command while not in a meeting.');
+            return null;
+        }
+
+        return meetingApi;
+    }
+
+    /**
+     * Invokes the {@code sendProxyConnectionEvent} method on the
+     * {@code JitsiMeetExternalApi} if it exists.
+     *
+     * @param {Object} event - The event to pass into the api.
+     * @private
+     * @returns {void}
+     */
+    _sendEventIfInMeeting(event) {
```

Comment:
this is a total bullshit comment, but for some reason this method is throwing me off.  i would've expected it to be inverted:
```
const meetingApi = this._getMeetingApiIfInMeeting();

if (!meetingApi) {
  logger.error('Failed to send proxy event.');
  return;
}

meetingApi.sendProxyConnectionEvent(event);
```

Reply:
you probably expected that because that's my normal style, and maybe a normal jitsi style(?) to handle errors first as the exception. I should change this.